torch/autograd/gradcheck.py

```diff
@@ -102,76 +114,132 @@ def get_stride(size):
             yield x_tensor, x_idx, d_idx
 
 
-def compute_gradient(fn, inputs, x, idx, delta, eps, is_mkldnn):
+def get_numerical_jacobian(fn, inputs, outputs=None, target=None, eps=1e-3,
```

Comment:
Hmm interesting. The one big repo that I see using this is  https://github.com/NVIDIA/MinkowskiEngine/blob/b40b98d2ee2f3f18c51ac609c0179482ffac071f/MinkowskiEngine/utils/gradcheck.py. 
Maybe we can just file an issue there.

I also see people using `get_analytical_jacobian` which we've already removed. So not sure if its worth it to maintain some kind of unofficial stable api for this type of thing since we'd have to also recreate how the old function returned `reentrant`, `grad_correct_dtype` etc.

Reply:
Updated the PR description to have a bc-breaking description. Also updated here as well https://github.com/pytorch/pytorch/pull/54049.

Reply:
Offline Alban expressed that this is reasonable but also asked if there's any way we can keep the old API lying around. We could potentially copy & paste the old API (since it's all python files) and have it go through a deprecation cycle?

Reply:
@zou3519 I added some wrapper functions that replicated the old API's behavior. The `get_numerical_jacobian` we use internally is now `_get_numerical_jacobian`. How does the deprecation message look?